modules/gitops-enable-replicas-for-argo-cd-server.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * argo-cd-custom-resource-properties.adoc
+
+:_content-type: PROCEDURE
+[id="enable-replicas-for-argo-cd-server_{context}"]
+= Enable replicas for Argo CD server and Repo server
+
+[role="_abstract"]
+You can enable a number of replicas for the Argo CD-server and Argo CD-repo-server workloads. As these workloads are stateless, the replica count can be increased to distribute the workload better among pods. However, if a horizontal autoscaler is enabled on Argo CD-server, it will be prioritized over the number set for replicas.
```

Comment:
```suggestion
You can enable a number of replicas for the Argo CD-server and Argo CD-repo-server workloads. Because these workloads are stateless, the replica count can be increased to distribute the workload better among pods. However, if a horizontal autoscaler is enabled on Argo CD-server, it will be prioritized over the number set for replicas.
```
- Excellent abstract. 
- When "as" means "because," use "because."

Reply:
Please use "Argo CD-server" or "Argo CD server" consistently. 
"Argo CD server" seems prevalent.